.github/workflows/sync-rest-api-docs.yaml

```diff
@@ -0,0 +1,49 @@
+name: Sync Camunda REST API
+
+on:
+  schedule:
+    - cron: "0 15 * * 4" # Runs every Thursday at 3 PM UTC
+  workflow_dispatch:
+
+jobs:
+  sync-file:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Camunda docs repository
+        uses: actions/checkout@v3
+        with:
+          repository: camunda/camunda-docs
+          path: docs
+
+      - name: Checkout Camunda repository
+        uses: actions/checkout@v3
+        with:
+          repository: camunda/camunda
+          path: rest-spec
+          sparse-checkout: zeebe/gateway-protocol/src/main/proto/rest-api.yaml
+          sparse-checkout-cone-mode: false
+
+      - name: Sync documentation
+        run: |
+          cd docs
+          mv ../rest-spec/zeebe/gateway-protocol/src/main/proto/rest-api.yaml api/camunda/camunda-openapi.yaml
+          npm i
+          npm run api:generate:camunda
+          if git diff-index --quiet HEAD; then
+            echo "No changes to commit."
+            exit 0
+          fi
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
+          path: docs
+          title: Update camunda REST API doc
+          body: This PR updates the camunda REST API
```

Comment:
Blocking: We will want more details in this body, especially to clearly describe the expected action. We don't necessarily have to figure out everything in this PR, but I think we should definitely add more context, and iterate it on the future. 

@akeller what do you think we should include in this body? Some things I'm thinking of: 

* More thorough description of what these changes represent, and how they were generated.
* Link back to the workflow that generated it 
* Clearly state a default urgency ("not urgent")
* Clarify that it only affects the "next" version of the docs

Reply:
Ideally, we get everything in the [PR template](https://github.com/camunda/camunda-docs/blob/main/.github/pull_request_template.md), which you captured above. We don't need the template filled out, just the info.

Reply:
Got it 👍